### PR TITLE
project ready for marking

### PR DESCRIPTION
**Pull request by nsharma-uk**
### Work Done
- 1 page  each of HTML and CSS code refactored.
- updated readme with all information and screenshots

### TODO
- [ ] Banner image has  title  attribute inside the <div> instead of alt text for image as unsure how to add alt text for image in css code, even after googling for a resolution.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
-# marketing-agency
\ No newline at end of file
+# Marketing-Agency
```
Comment (LeeFarnell) [inline README.md:1]: This is a brilliant ReadMe. Well done!

File: assets/css/style.css [hunk 2]
```diff
@@ -0,0 +1,134 @@
+* {
```
Comment (LeeFarnell) [inline assets/css/style.css:1]: As mentioned in your feedback, this file needs comments. They can be as simple as 'Main Section styling' etc, please add these in.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon Marketing</title>
+  </head>
+
+  <body>
+<!--header section including navbar-->
+    <div class="header">
```
Comment (LeeFarnell) [inline index.html:11]: This Div should of been a Header tag which would make it more semantic. Please consider changing this.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+          </li>
+        </ul>
+      </div>
+    </div>
+
+<!--Banner image-->
+    <div class="banner-img" title="Marketing meeting."></div>
+    
+<!-- 3 sections with features of marketing company-->
+    <div class="content">
```
Comment (LeeFarnell) [inline index.html:34]: This Div should of been a Main tag which would make it more semantic. Please consider changing this.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+          for the right business as they travel, shop, or sit on their couch at
+          home. Search Engine Optimization (SEO) allows you to increase your
+          visibility and find the right customers for your business.
+        </p>
+      </div>
+    </section> 
+   
+
+    <section>
+      <div
```
Comment (LeeFarnell) [inline index.html:57]: As above, no need for divs in section tag

File: index.html [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+          media allows anyone with an internet connection to say whatever they
+          want about your business. Online Reputation Management gives you the
+          control over what potential customers see when they search for your
+          business.
+        </p>
+      </div>
+    </section>
+
+    <section>
+      <div id="social-media-marketing" class="features">
```
Comment (LeeFarnell) [inline index.html:78]: As above, no need for divs in section tag

File: index.html [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+          to your brand, using analytics to find the right markets and increase
+          your lead generation.
+        </p>
+      </div>
+    </div>
+  </section>
+
+<!--Side bar with benefits info-->
+<div class="benefits">
+  <div class="benefit1">
```
Comment (LeeFarnell) [inline index.html:97]: The divs inside of this should be sections to make it more semantic.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+    <h3>Lead Generation</h3>
+    <img src="./assets/images/lead-generation.png" alt="Logo of cogs being funnelled into dollar signs."  />
+    <p>
+      Inbound strategies for lead generation require less work for your
+      business, bringing customers directly to your website.
+    </p>
+  </div>
+  <div class="benefit1">
```
Comment (LeeFarnell) [inline index.html:105]: As above, this div should be a section tag.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+    <h3>Brand Awareness</h3>
+    <img src="./assets/images/brand-awareness.png" alt="Logo of lightbulb with wireless icon." />
+    <p>
+      Users find your business through paid and organic searches, increasing
+      the search ranking and visibility for your business.
+    </p>
+  </div>
+  <div class="benefit1">
```
Comment (LeeFarnell) [inline index.html:113]: As above, this div should be a section tag.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+    <img src="./assets/images/cost-management.png" alt="Logo of a cog with dollar signs under." />
+    <p>
+      As the search ranking for your business increases, your advertising
+      costs decrease, and you no longer need to advertise your page.
+    </p>
+  </div>
+</div>
+
+<!--webpage footer-->   
+<div class="footer">
```
Comment (LeeFarnell) [inline index.html:124]: This div could be a footer tag, it would be more semantic

File: README.md (hunk 1 above)
Comment (nsharma-uk) [reply README.md:1]: Thanks. That's a good start. :)

**Review by nsharma-uk — COMMENTED** [review]
All finished

**Pull request closed without merging**